objectivec/GPBDescriptor.h

```diff
@@ -81,6 +81,16 @@ typedef NS_ENUM(uint8_t, GPBFieldType) {
 @property(nonatomic, readonly, getter=isWireFormat) BOOL wireFormat;
 /** The class of this message. */
 @property(nonatomic, readonly) Class messageClass;
+/**
+ * If this message was nested under another message, it will return
+ * that containing descriptor.
+ */
+@property(readonly, nullable) GPBDescriptor *containingType;
```

Comment:
/** Containing message descriptor if this message is nested, or nil otherwise. */